tests/sku_create_test.py

```diff
@@ -3,6 +3,7 @@
 import pytest
 import subprocess
 import re
+import shutil
```

Comment:
Can you please organize these per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports), and alphabetize in each group? E.g.,
```python
import os
import re
import shutil
import subprocess
import sys

import pytest
```

Reply:
Addressed in next commit.